scripts/solr_reindex_template.py

```diff
@@ -0,0 +1,40 @@
+import typing
```

Comment:
@datadavev new file designed to be a starting point for future reindex/record modification operations…